### PR TITLE
[Feature] Add playground module

### PR DESCRIPTION
**Pull request by labbomb**
image

### Purpose
Please refer to the main `issue`  [#44](https://github.com/apache/incubator-paimon-webui/issues/44)

### CODE REVIEW

File: paimon-web-ui-new/pnpm-lock.yaml [hunk 1]
```diff
@@ -1393,7 +1399,6 @@ packages:
 
   /argparse@2.0.1:
     resolution: {integrity: sha512-8+9WqebbFzpX9OR+Wa6O29asIogeRMzcGtAINdpMHHyAg10f05aSFVBbcEqGf/PXw1EjAZ+q2/bEBg3DvurK3Q==}
-    dev: true
```
Comment (SteNicholas) [inline paimon-web-ui-new/pnpm-lock.yaml:1396]: Is it better to set `dev` to false?

File: paimon-web-ui-new/src/locales/en/modules/playground.ts [hunk 2]
```diff
@@ -16,6 +16,22 @@ specific language governing permissions and limitations
 under the License. */
 
 export default {
-  select_catalog: 'Select Catalog',
-  search: 'Search'
+  database_query: 'Database Query',
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/locales/en/modules/playground.ts:19]:
```suggestion
  query: 'Query',
```

File: paimon-web-ui-new/src/locales/zh/modules/playground.ts [hunk 3]
```diff
@@ -16,6 +16,22 @@ specific language governing permissions and limitations
 under the License. */
 
 export default {
-  select_catalog: '选择 Catalog',
-  search: '搜索'
+  database_query: '数据库查询',
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/locales/zh/modules/playground.ts:19]:
```suggestion
  query: '查询',
```

File: paimon-web-ui-new/src/locales/zh/modules/playground.ts [hunk 4]
```diff
@@ -16,6 +16,22 @@ specific language governing permissions and limitations
...
+  data: '数据',
+  saved_query: '已保存查询',
+  query_record: '查询记录',
+  search: '搜索',
+  run: '运行',
+  format: '格式化',
+  save: '保存',
+  clear: '清空',
+  unfold: '展开',
+  pack_up: '收起',
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/locales/zh/modules/playground.ts:34]:
```suggestion
  collapse: '折叠',
```

File: paimon-web-ui-new/src/locales/en/modules/playground.ts [hunk 5]
```diff
@@ -16,6 +16,22 @@ specific language governing permissions and limitations
...
+  data: 'Data',
+  saved_query: 'Saved Query',
+  query_record: 'Query Record',
+  search: 'Search',
+  run: 'Run',
+  format: 'Format',
+  save: 'Save',
+  clear: 'Clear',
+  unfold: 'Unfold',
+  pack_up: 'Pack Up',
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/locales/en/modules/playground.ts:34]:
```suggestion
  collapse: 'Collapse',
```

File: paimon-web-ui-new/src/router/modules/playground.ts [hunk 6]
```diff
@@ -27,7 +27,16 @@ export default [
...
         name: 'playground',
         meta: { title: '查询控制台' },
-        component: () => import('@/views/playground')
+        redirect: { name: 'playground-database' },
+        component: () => import('@/views/playground'),
+        children: [
+          {
+            path: '/playground/database',
+            name: 'playground-database',
+            meta: { title: '数据库查询' },
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/router/modules/playground.ts:36]:
```suggestion
            meta: { title: '查询' },
```

File: paimon-web-ui-new/src/views/playground/components/database/components/console/index.tsx [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+                    text
+                    onClick={this.handleDown}
+                    v-slots={{
+                      icon: () => <n-icon component={ChevronDown}></n-icon>
+                    }}
+                  >
+                  </n-button>
+                )
+              }}>
+              <span>{this.t('playground.pack_up')}</span>
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/views/playground/components/database/components/console/index.tsx:102]:
```suggestion
              <span>{this.t('playground.collapse')}</span>
```

File: paimon-web-ui-new/src/locales/en/modules/playground.ts [hunk 8]
```diff
@@ -16,6 +16,22 @@ specific language governing permissions and limitations
...
 
 export default {
-  select_catalog: 'Select Catalog',
-  search: 'Search'
+  database_query: 'Database Query',
+  workbench: 'Workbench',
+  task_operation_and_maintenance: 'Task Operation and Maintenance',
+  settings: 'Settings',
+  terminal: 'Terminal',
+  branch: 'Branch',
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/locales/en/modules/playground.ts:24]:
```suggestion
  git_branch: 'Git Branch',
```

File: paimon-web-ui-new/src/locales/zh/modules/playground.ts [hunk 9]
```diff
@@ -16,6 +16,22 @@ specific language governing permissions and limitations
...
 
 export default {
-  select_catalog: '选择 Catalog',
-  search: '搜索'
+  database_query: '数据库查询',
+  workbench: '工作台',
+  task_operation_and_maintenance: '任务运维',
+  settings: '设置',
+  terminal: '终端',
+  branch: '分支',
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/locales/zh/modules/playground.ts:24]:
```suggestion
  git_branch: 'Git 分支',
```

**Review by SteNicholas — CHANGES_REQUESTED** [review]
@labbomb, thanks for contribution. I left some minor comments for this pull request. PTAL.

File: paimon-web-ui-new/pnpm-lock.yaml (hunk 1 above)
Comment (labbomb) [reply paimon-web-ui-new/pnpm-lock.yaml:1396]: No impact, do not adjust

**Review by SteNicholas — APPROVED** [review]
LGTM. Thanks for @labbomb updates.

**Pull request merged**